_data/modules.yml

```diff
@@ -0,0 +1,15 @@
+- name: "AR"
```

Comment:
Shouldn't this file be generated during the deployment from autils repo?

Reply:
New workflow created that takes this into account and applies avatars.